helx/charts/appstore/templates/configmap.yaml

```diff
@@ -30,5 +30,5 @@ data:
   TEST_USERS_PATH: "{{ .Values.django.TEST_USERS_PATH }}"
   REACT_APP_SEMANTIC_SEARCH_ENABLED: "{{ .Values.helx_ui.REACT_APP_SEMANTIC_SEARCH_ENABLED }}"
   REACT_APP_WORKSPACES_ENABLED: "{{ .Values.helx_ui.REACT_APP_WORKSPACES_ENABLED }}"
-  REACT_APP_HELX_SEARCH_URL: '{{ .Values.helx_ui.REACT_APP_HELX_SEARCH_URL }}'
-  REACT_APP_UI_BRAND_NAME: {{ .Values.helx_ui.REACT_APP_UI_BRAND_NAME }}
+  REACT_APP_HELX_SEARCH_URL: "{{ .Values.helx_ui.REACT_APP_HELX_SEARCH_URL }}"
```

Comment:
This actually needs to be single-quoted. The URL is used in a sed script and we want to preserve the escaping of the '/' character, if we use double quotes it will create an invalid escape sequence